meal-sharing/Program.cs

```diff
@@ -0,0 +1,54 @@
+using Microsoft.AspNetCore.Mvc;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddScoped<IMealService, FileMealService>();
+var app = builder.Build();
+
+app.MapGet("/", ([FromServices] IMealService mealSharingService) =>
+{
+    var result = mealSharingService.ListMeals();
+    return result;
+});
+
+app.MapPost("/", ([FromServices] IMealService mealSharingService, Meal meal) =>
+{
+    mealSharingService.AddMeal(meal);
+});
+
+app.Run();
+
+public interface IMealService
+{
+    void AddMeal(Meal meal);
+    List<Meal> ListMeals();
+}
+public class Meal
+{
+    public string Headline { get; set; }
+    public string Url { get; set; }
+    public string Body { get; set; }
+    public string Location { get; set; }
+    public int Price { get; set; }
```

Comment:
I would maybe argue that Price is `decimal` and not an `integer` but other than that it looks great.